spec/acceptance/v1/tickets_spec.rb

```diff
@@ -10,8 +10,9 @@
 	let!(:category) { FactoryBot.create(:category, name: "Hardware", priority: 0, department_id: department.id)}
 	let!(:role) { FactoryBot.create(:role, name: 'super_admin')}
 	let!(:role1) { FactoryBot.create(:role, name: 'employee')}
-	let(:user) { FactoryBot.create(:user, role_id: role.id, department_id: department.id) }
-	let(:user1) { FactoryBot.create(:user, role_id: role.id) }
+	let(:user) { FactoryBot.create(:user, role_id: role.id, email: "abcd@gmail.com") }
+	let(:user1) { FactoryBot.create(:user, role_id: role.id, email: "pqrs@gmail.com") }
```

Comment:
Is there any issue faced in the spec for for using Faker? - we have hard coded emails for creating user - we can use the factory right?

Reply:
Yes but there is validation for using same email @varungadde99